app/components/header.tsx

```diff
@@ -93,7 +92,7 @@ export default function Header() {
 
       <Flex justifyContent="flex-end" alignItems="center" color="white" gap="5" width="100%">
         <Hide below="lg">
-          <Text id="header-user">{user.username}</Text>
+          <Text id="header-user">{user ? user.username : ''}</Text>
```

Comment:
```ts
{ user && <Text id="header-user">{user.username}</Text> }
```

Reply:
I am a bit confused as to why we make the user optional? What is the purpose of this if you don't mind me asking?

Reply:
The session and the user info the header relies on will be deleted when the SLO logout prompt is displayed, but the header is still rendered.